packages/synapse-react-client/src/components/TimelinePlot/TimelinePhase.tsx

```diff
@@ -260,23 +284,19 @@ const TimelinePhase = ({
           setHoverEvent(undefined)
         }}
       />
-      {selectedRows && (
-        <Dialog
-          onClose={() => setClickEvent(undefined)}
-          open={!!selectedRows && selectedRows.length > 0}
-        >
-          {selectedRows.map(row => {
-            return (
-              <ObservationCard
-                key={row.rowId}
-                data={row.values}
-                schema={schema}
-                includePortalCardClass={false}
-              />
-            )
-          })}
-        </Dialog>
-      )}
+      <Dialog
+        onClose={() => setClickEvent(undefined)}
+        open={selectedRows && selectedRows.length > 0}
+      >
+        <DialogContent>
+          <CardContainerLogic
+            filterColumnName="observationType"
+            sql={sql}
+            type={OBSERVATION_CARD}
+            initialLimit={3}
+          />
```

Comment:
We can add additional filters using the `searchParams` prop (which needs a better name, but that would be a massive change).

We add the `sqlOperator` prop to tell our code to add the searchParams as a filter with the `IN` clause.

And we can also add the `lockedColumn` to hide any UI that would indicate the ROW_ID filter (it may not make a difference now, but we can add it to be safe)

```suggestion
          <CardContainerLogic
            filterColumnName="observationType"
            sql={observationsSql}
            searchParams={{["ROW_ID"]: selectedRowIds}}
            sqlOperator="IN"
            lockedColumn={{ columnName: "ROW_ID" }}
            type={OBSERVATION_CARD}
            initialLimit={3}
          />
```